src/Rokt-Kit.js

```diff
@@ -85,6 +88,13 @@ var constructor = function () {
             placementEventMapping
         );
 
+        // Set dynamic OTHER_IDENTITY based on server settings
+        // Convert to lowercase since server sends TitleCase (e.g., 'Other' -> 'other')
+        if (settings.hashedEmailUserIdentityType) {
+            MAPPED_EMAIL_SHA256_IDENTITY =
```

Comment:
I would make `MAPPED_EMAIL_SHA256_IDENTITY` a standard variable if you plan on assigning it to a value. This looks like a constant as is, so it shouldn't be assigned a value like this.